docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+##### tablet_stat_update_interval_second
+
+- **Default:** 300
+- **Description:** The time interval at which the FE retrieves tablet statistics from each BE. Unit: second.
+
+#### StarRocks shared-data cluster
+
+##### run_mode
+
+- **Default**: shared_nothing
+- **Description**: The running mode of the StarRocks cluster. Valid values: shared_data and shared_nothing (Default).
+
+shared_data indicates running StarRocks in shared-data mode.
+shared_nothing indicates running StarRocks in shared-nothing mode.
+CAUTION
+You cannot adopt the shared_data and shared_nothing modes simultaneously for a StarRocks cluster. Mixed deployment is not supported.
+DO NOT change run_mode after the cluster is deployed. Otherwise, the cluster fails to restart. The transformation from a shared-nothing cluster to a shared-data cluster or vice versa is not supported.
+
+##### cloud_native_meta_port
+
+- **Default**: 6090
+- **Description**: The cloud-native meta service RPC port. Default: 6090.
+
+##### cloud_native_storage_type
+
+- **Default**: S3
+- **Description**: The type of object storage you use. In shared-data mode, StarRocks supports storing data in Azure Blob (supported from v3.1.1 onwards), and object storages that are compatible with the S3 protocol (such as AWS S3, Google GCP, and MinIO). Valid value: S3 (Default) and AZBLOB. If you specify this parameter as S3, you must add the parameters prefixed by aws_s3. If you specify this parameter as AZBLOB, you must add the parameters prefixed by azure_blob.
+
+##### aws_s3_path
+
+- **Default**: N/A
+- **Description**: The S3 path used to store data. It consists of the name of your S3 bucket and the sub-path (if any) under it, for example, testbucket/subpath.
+
+##### aws_s3_endpoint
+
+- **Default**: N/A
+- **Description**: The endpoint used to access your S3 bucket, for example, `https://s3.us-west-2.amazonaws.com`.
+
+##### aws_s3_region
+
+- **Default**: N/A
+- **Description**: The region in which your S3 bucket resides, for example, us-west-2.
+
+##### aws_s3_use_aws_sdk_default_behavior
+
+- **Default**: false
+- **Description**: Whether to use the default authentication credential of AWS SDK. Valid values: true and false (Default).
+
+##### aws_s3_use_instance_profile
+
+- **Default**: false
+- **Description**: Whether to use Instance Profile and Assumed Role as credential methods for accessing S3. Valid values: true and false (Default).
+  - If you use IAM user-based credential (Access Key and Secret Key) to access S3, you must specify this item as false, and specify aws_s3_access_key and aws_s3_secret_key.
+  - If you use Instance Profile to access S3, you must specify this item as true.
+  - If you use Assumed Role to access S3, you must specify this item as true, and specify aws_s3_iam_role_arn.
+  - And if you use an external AWS account, you must also specify aws_s3_external_id.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/846be8fb-7c56-4d78-a9e6-adb7ef57cab8)
I suggest wrapping all these in code format